refinery/core/models.py

```diff
@@ -2221,6 +2221,10 @@ class SiteProfile(models.Model):
 
     site = models.OneToOneField(Site, related_name='profile')
     repo_mode_home_page_html = models.TextField(blank=True)
+    about_markdown = models.TextField(blank=True)
+    intro_markdown = models.TextField(blank=True)
+    twitter_username = models.CharField(max_length=100, blank=True)
+    yt_videos = models.TextField(blank=True)
```

Comment:
Maybe `youtube_videos` here instead?

Also rather than a `TextField` it looks like `yt_videos` could be better expressed through a relationship to another model? 

Alternatives to that ^^^ would be:
- a"pickle-able" field (requires a [third party library](https://pypi.org/project/django-picklefield/)) 
- a [JSONField](https://docs.djangoproject.com/en/1.9/ref/contrib/postgres/fields/#jsonfield) (requires django >= 1.9 and is PostgreSQL specific)